package.json

```diff
@@ -58,6 +58,7 @@
   "dependencies": {
     "@backstage/repo-tools": "^0.13.3",
     "@useoptic/optic": "^1.0.5",
+    "geoserver-rest-client": "https://github.com/sweco-se1g7y/geoserver-node-client.git",
```

Comment:
Should be in `plugins/catalog-backend-module-geoserver-data/package.json` (same goes for pg-listen as well I just realized)